docs/manual/common/guide/production/LightbendPlatform.md

```diff
@@ -37,8 +37,8 @@ See [Integrating Lagom with Lightbend Telemetry](https://github.com/lagom/lagom-
 
 Bintray credentials are required to build applications using the Lightbend Platform. Lightbend customers should log into the [support portal](https://portal.lightbend.com/ReactivePlatform/EnterpriseSuiteCredentials) to obtain their credentials. Follow the links below to see how to supply the credentials when using sbt or Maven.
 
-* [Lightbend Platform setup for sbt](https://developer.lightbend.com/docs/reactive-platform/2.0/setup/setup-sbt.html)
-* [Lightbend Platform setup for Maven](https://developer.lightbend.com/docs/reactive-platform/2.0/setup/setup-maven.html)
+* [Lightbend Platform setup for sbt](https://developer.lightbend.com/docs/lightbend-platform/2.0/setup/setup-maven.html)
```

Comment:
Oops, I just noticed this is one `maven` too many!

Reply:
I'll fix in the `1.4.x` back port pull request (#1836) and the `master` forward port (#1837) and then send another one to fix 1.5.x